lib/hammer_cli/settings.rb

```diff
@@ -20,7 +20,7 @@ def self.load_from_paths(files)
           load_from_file(File.join(full_path, 'cli_config.yml'))
           load_from_file(File.join(full_path, 'defaults.yml'))
           # load config for modules
-          Dir.glob(File.join(full_path, 'cli.modules.d/*.yml')).sort.each do |f|
+          Dir.glob(File.join(full_path, '**/*.yml')).sort.each do |f|
```

Comment:
It's a great chance to change the `load_from_paths` method to also accept non-arrays (e.g. string containing just one path). Can you do this?